src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -324,6 +314,29 @@ class ReviewCollapsibleChapter extends React.Component {
     );
   };
 
+  focusOnPage = key => {
+    const name = `${key.replace(/:/g, '\\:')}`;
+    const scrollElement = document.querySelector(
+      `[name="${name}ScrollElement"]`,
+    );
+
+    if (scrollElement && scrollElement.parentElement) {
+      // Wait for edit view to render
+      setTimeout(() => {
+        const focusableElements = getFocusableElements(
+          scrollElement.parentElement,
+        );
+
+        // Sets focus on the first focusable element
+        focusOnChange(name, `[id="${focusableElements[0].id}"]`);
+      }, 0);
+    }
+  };
+
+  scrollToPage = key => {
+    scroller.scrollTo(`${key}ScrollElement`, getScrollOptions({ offset: -40 }));
```

Comment:
Could you put `-40` in a constant at the top of the file instead of inlining it here? Something like `const OFFSET = -40`

Reply:
Great idea!